CONTRIBUTING.md

```diff
@@ -82,3 +90,55 @@ export class MyConstruct extends Construct {
   * Other import methods normally require access to the context API which is beyond the scope of these constructs. We'd like to address this in the future.
 * Constructs should implement an interface so that the import methods (e.g. `.fromSomethingAttributes()`) can return the interface type.
 
+
+
+## Testing
+
+Tests are fundamental to building good constructs. The following testing guidelines should be followed:
+
+* All tests should be stored in the `test/aws-<something>` subdirectory. For example, if you are creating
```

Comment:
```suggestion
* All tests should be stored in the `test/<something>` subdirectory. For example, if you are creating
```
Constructs outside of AWS or using multiple AWS services don't need to start with aws

Reply:
That's true, but I assume that most of the constructs people will be creating for this library will be L2s on specific AWS services so figured the 'aws' prefix was appropriate.

Reply:
Interestingly, while this might be true initially, I assume we will have more non-AWS constructs in the future. I really hope we can create meaningful L3s (patterns).

Reply:
I'll add something to distinguish between the two.